packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -45,6 +42,24 @@ const propTypes = {
    * Aligns the menu to the start of the container
    */
   isStartAligned: PropTypes.bool,
+
+  /**
+   * @private
+   * Switches to using horizontal ellipses instead of vertical ones when menu items are truncated.
+   */
+  useHorizontalIcon: PropTypes.bool,
```

Comment:
If we have this prop, why did the screen shots change for the ellipsis?  Shouldn't that only happen for your FilePath test?

Reply:
This was a bug discovered when I was working on this component, all themes other than default was rendering the wrong icon. This PR also fixes it to be consistent across all themes